### PR TITLE
Create mips.xml

### PR DESCRIPTION
**Pull request by 0x3h**
this is a configuration file for the mips architecture, it does not include all instructions but the most of them.

### CODE REVIEW

File: Configurations/mips.xml [hunk 1]
```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<documentation>
+	<document id="mips32" name="MIPS Architecture Reference Manual" version="A.k">
```
Comment (alexhude) [inline Configurations/mips.xml:3]: If there is a version or revision for this document, could you please add it to `version=""`?

**Review by alexhude — CHANGES_REQUESTED** [review]
Thanks for this config. Just a tiny bit to fix and I will merge it.

File: Configurations/mips.xml (hunk 1 above)
Comment (0x3h) [reply Configurations/mips.xml:3]: there is no version, i used http://www.mrc.uidaho.edu/mrc/people/jff/digital/MIPSir.html as a reference.

Comment (alexhude) [reply Configurations/mips.xml:3]: Hm, in this case I will probably update code to support links in the <path>

**Pull request merged**